src/hit/api/evaluator.cpp

```diff
@@ -388,4 +402,7 @@ namespace hit {
     void CKKSEvaluator::rescale_to_next_inplace_internal(CKKSCiphertext &){};
     void CKKSEvaluator::relinearize_inplace_internal(CKKSCiphertext &){};
     void CKKSEvaluator::print_stats(const CKKSCiphertext &){};
+    CKKSCiphertext CKKSEvaluator::bootstrap_internal(const CKKSCiphertext &ct, bool) {
+        return ct;
+    };
```

Comment:
Nit: Missed parameter name `rescale_for_bootstrapping`?
```
virtual CKKSCiphertext bootstrap_internal(const CKKSCiphertext &ct, bool rescale_for_bootstrapping);
```

Reply:
There is a compiler warning or clang-tidy warning about unused parameters, so I've removed the parameter name to remove the warning.